packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1208,13 +1216,32 @@ export default class NewVm extends BaseComponent {
               </span>
             </LineItem>
             <br />
-            <DebounceTextarea
-              className='form-control'
-              disabled={installMethod !== 'customConfig'}
-              onChange={this._linkState('customConfig')}
-              rows={7}
-              value={defined(customConfig, DEFAULT_CLOUD_CONFIG_TEMPLATE)}
-            />
+            <Row>
+              <Col size={3}>
+                <div className='text-muted'>User config</div>
+                <DebounceTextarea
+                  className='form-control'
+                  disabled={installMethod !== 'customConfig'}
+                  onChange={this._linkState('customConfig')}
+                  rows={7}
+                  value={defined(customConfig, DEFAULT_CLOUD_CONFIG_TEMPLATE)}
+                />
+              </Col>
+              <Col size={3}>
+                <div className='text-muted'>Network config</div>
```

Comment:
Should we let users store their customized network configs?

Reply:
It would be nice, but I think we should stay as simple as possible for the first implementation.

Reply:
I agree :+1: